Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -84,7 +84,11 @@ protected override void Awake()
             }
             else
             {
-                XRDevice.SetTrackingSpaceType(transparentTrackingSpaceType);
+                // Removed for now, until the HoloLens tracking space type story is more clear.
+                //XRDevice.SetTrackingSpaceType(transparentTrackingSpaceType);
+
+                enabled = false;
```

Comment:
Should we destroy the component instead of disabling, like we do in other scripts?

Reply:
My hesitation to destroy was to preserve the Singleton instance, but if devs are properly checking `IsInitialized` and whatnot, that should be okay.

Reply:
That's a good point as well.